types/robust-point-in-polygon/robust-point-in-polygon-tests.ts

```diff
@@ -0,0 +1,4 @@
+import assert = require('assert');
+import inPolygon = require('robust-point-in-polygon');
+
+assert.equal(typeof inPolygon, 'function');
```

Comment:
This isn't test that's appropriate for DefinitelyTyped. Just write code for how a user is supposed to use this library.

Reply:
I've updated the tests